src/lib/components/Mission.svelte

```diff
@@ -1,125 +1,106 @@
-								/>
-							</svg>
-						</div>
-						SSL certificates
-					</dt>
-					<dd class="mt-2 text-base leading-7 text-gray-600">
-						Sit quis amet rutrum tellus ullamcorper ultricies libero dolor eget. Sem sodales gravida
-						quam turpis enim lacus amet.
-					</dd>
-				</div>
-				<div class="relative pl-16">
-					<dt class="text-base font-semibold leading-7 text-gray-900">
-						<div
-							class="absolute left-0 top-0 flex h-10 w-10 items-center justify-center rounded-lg bg-indigo-600"
-						>
-							<svg
-								class="h-6 w-6 text-white"
-								fill="none"
-								viewBox="0 0 24 24"
-								stroke-width="1.5"
-								stroke="currentColor"
-								aria-hidden="true"
-							>
-								<path
-									stroke-linecap="round"
-									stroke-linejoin="round"
-									d="M16.023 9.348h4.992v-.001M2.985 19.644v-4.992m0 0h4.992m-4.993 0l3.181 3.183a8.25 8.25 0 0013.803-3.7M4.031 9.865a8.25 8.25 0 0113.803-3.7l3.181 3.182m0-4.991v4.99"
-								/>
-							</svg>
-						</div>
-						Simple queues
-					</dt>
-					<dd class="mt-2 text-base leading-7 text-gray-600">
-						Quisque est vel vulputate cursus. Risus proin diam nunc commodo. Lobortis auctor congue
-						commodo diam neque.
-					</dd>
-				</div>
-				<div class="relative pl-16">
-					<dt class="text-base font-semibold leading-7 text-gray-900">
-						<div
-							class="absolute left-0 top-0 flex h-10 w-10 items-center justify-center rounded-lg bg-indigo-600"
-						>
-							<svg
-								class="h-6 w-6 text-white"
-								fill="none"
-								viewBox="0 0 24 24"
-								stroke-width="1.5"
-								stroke="currentColor"
-								aria-hidden="true"
-							>
-								<path
-									stroke-linecap="round"
-									stroke-linejoin="round"
-									d="M7.864 4.243A7.5 7.5 0 0119.5 10.5c0 2.92-.556 5.709-1.568 8.268M5.742 6.364A7.465 7.465 0 004.5 10.5a7.464 7.464 0 01-1.15 3.993m1.989 3.559A11.209 11.209 0 008.25 10.5a3.75 3.75 0 117.5 0c0 .527-.021 1.049-.064 1.565M12 10.5a14.94 14.94 0 01-3.6 9.75m6.633-4.596a18.666 18.666 0 01-2.485 5.33"
-								/>
-							</svg>
-						</div>
-						Advanced security
-					</dt>
-					<dd class="mt-2 text-base leading-7 text-gray-600">
-						Arcu egestas dolor vel iaculis in ipsum mauris. Tincidunt mattis aliquet hac quis. Id
-						hac maecenas ac donec pharetra eget.
-					</dd>
-				</div>
-			</dl>
-		</div>
+
+<SectionHeader title="FREE COURSES" description="You can check out some of our free courses" />
+<div class="container">
+	<div class="buttons">
+		<button
+			on:click={() => setActiveTab('Mission')}
+			class = "tab-button"
+			class:active={activeTab === 'Mission'}
+		>
+			Mission
+		</button>
+		<button
+			on:click={() => setActiveTab('Vision')}
+			class = "tab-button"
+			class:active={activeTab === 'Vision'}
+		>
+			Vision
+		</button>
+	</div>
+	<br>
+	<div>
+		{#if activeTab === 'Mission'}
+		  {#each mission as item (item.id)}
```

Comment:
by removing mission and vision inside the array you can just directly use mission.id and mission.description, same comment for vision